README.md

```diff
@@ -208,7 +208,7 @@ gh_pr
 ```
 
 - Merge to dev
-- [create PR to merge to main](https://github.com/UN-OCHA/numbers-site/compare/main...develop)
+- [create PR to merge to main](https://github.com/UN-OCHA/ocha-api-site/compare/main...develop)
 - Merge to main
 - [Tag a new release](./gh_release)
```

Comment:
I know this isn't part of the changes made, but is there a script missing?

Reply:
Script is on my local, source can be found at https://tips.attiks.com/2022/03/29/generate-github-release-link/

Reply:
Got it. Thanks. Might be good to add a comment with that link.